README.md

```diff
@@ -1910,6 +1910,9 @@ sudo sysdiagnose -f ~/Desktop/
 
 #### Create Bootable Installer
 ```sh
+# macOS 11 (Big Sur)
+sudo /Applications/Install\ macOS\ Big\ Sur.app/Contents/Resources/createinstallmedia --volume /Volumes/USB
```

Comment:
Why did you leave `--nointeraction --downloadassets` off? All other entries have them. Please verify that they still exist or do not exist, update the pull request  and add that information to the pull request description.